add Missing note: The file should be recompiled using Visual Studio for the changes to take effect (#11391)
flutter/website: src/content/platform-integration/platform-channels.md

You should now be able to run the application on Windows.
If your device doesn't have a battery,
it displays 'Battery level not available'.
:::note
To ensure that the changes take effect, recompile the project.
Otherwise, you might encounter the following error: **Failed to get 

> battery level: Memory leak.**

Reviewer: I don't see how it's possible to get this error message string from the provided example code, so this needs further debugging with repro steps; changing docs is premature (especially since the step the new text is recommending should not be necessary).

Author: > I don't see how it's possible to get this error message string from the provided example code, so this needs further debugging with repro steps; changing docs is premature (especially since the step the new text is recommending should not be necessary).

Should i open an issue with repro steps first ?

You should now be able to run the application on Windows.
If your device doesn't have a battery,
it displays 'Battery level not available'.

> Note: To ensure the changes take effect, the project should be recompiled 
> with Visual Studio.Otherwise, you may encounter an error saying: Failed to get 
> battery level: Memory leak .

Reviewer 2: ```suggestion
:::note
To ensure that the changes take effect, recompile the project.
Otherwise, you might encounter the following error: **Failed to get 
battery level: Memory leak.**
:::
```
